app/src/main/java/com/bachelor_group54/funnregistrering/Funn.java

```diff
@@ -1,17 +1,27 @@
 package com.bachelor_group54.funnregistrering;
 
-import android.graphics.Bitmap;
-
 import java.io.Serializable;
 
-public class Funn implements Serializable { //Klasse med attributter, getters og setters for funn.
+//Class for holding info about each found (Contains variables with getters and setters )
+public class Funn implements Serializable {
     private String tittel, grunneierNavn, grunneierAdresse, grunneierPostNr, grunneierPostSted,
             grunneierTlf, grunneierEpost, funnsted, kommune, fylke, gjenstand, gjenstandMerking,
-            datum, arealType, beskrivelse, funndato, dato;
+            datum, arealType, beskrivelse, funndato, dato, opplysninger;
+
+    //Sets the depth to -1, then we can check if its not set later, as negative values should not be a valid input
+    //The maximum latitude value is 90 and the maximum longitude value is 180. Setting both to 200 means we can later check if the value has been set or not
```

Comment:
Aight, så det er for å se om de blir satt eller ikke. Kan se bort i fra kommenten på dette i den andre branchen.